torchinductor/codegen/autotuner.py

```diff
@@ -0,0 +1,166 @@
+import builtins
+
+import torch
+import triton
+
+import torchinductor
+import torchinductor.triton_ops
+
+aten = torch.ops.aten
+triton_ops = torchinductor.triton_ops
+
+# Instatiatie Autotuner class object -- init the cache, and sort of global
+
+# Autotuning an op
+#   1) Register that Node in the codegen of ExternKernelOP
+#   2) Autotuner
+#   1) register that IRNode "ExternKerneNode" in Autotuner - autotune()
```

Comment:
```suggestion
#   3) register that IRNode "ExternKerneNode" in Autotuner - autotune()
```

Reply:
I don't totally understand these steps

Reply:
Sorry about the confusion, these comments are not related to current code and should be deleted